src/SFA.DAS.EAS.Web/ViewModels/TransactionDownloadViewModel.cs

```diff
@@ -0,0 +1,57 @@
+﻿using SFA.DAS.EAS.Application.Messages;
+using SFA.DAS.EAS.Application.Queries.GetTransactionsDownloadResultViewModel;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+
+namespace SFA.DAS.EAS.Web.ViewModels
+{
+    public class TransactionDownloadViewModel : ViewModel<GetTransactionsDownloadQuery>
+    {
+        public string AccountHashedId { get; set; }
+
+        public TransactionsDownloadStartDateMonthYearDateTime StartDate
+        {
+            get
+            {
+                if (Message != null)
+                {
+                    return Message.StartDate;
+                }
+
+                return null;
+            }
+            set
+            {
+                if (Message == null)
+                {
+                    Message = new GetTransactionsDownloadQuery();
```

Comment:
Because you're inheriting from `ViewModel<T>` this will be created automatically for you.